S2VX.Game.Tests/VisualTests/EndGameScreenTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Screens;
+using osu.Framework.Testing;
+using S2VX.Game.EndGame;
+using S2VX.Game.Play;
+using S2VX.Game.SongSelection;
+using System.IO;
+
+namespace S2VX.Game.Tests.VisualTests {
+    public class EndGameScreenTests : S2VXTestScene {
+        [Resolved]
+        private AudioManager Audio { get; set; }
+
+        [Cached]
+        private ScreenStack ScreenStack { get; } = new();
+        private SongSelectionScreen SongSelectionScreen { get; } = new();
+
+        [BackgroundDependencyLoader]
+        private void Load() {
+            ScreenStack.Push(SongSelectionScreen);
+            Add(ScreenStack);
+        }
+
+        [SetUpSteps]
+        private void SetUpSteps() => AddStep("Reset to song selection screen", () => SongSelectionScreen.MakeCurrent());
```

Comment:
Does this implicitly remove any screens on top?

Reply:
Yeah, I confirmed through the Ctrl+F1 menu, also the description should be fairly clear:
![image](https://user-images.githubusercontent.com/3955187/123741392-d65ee200-d85e-11eb-9fe4-a6515d435d07.png)